README.md

```diff
@@ -105,14 +105,9 @@ If you prefer to use the built-in IPSEC VPN on Apple devices, or need "Connect o
 
 WireGuard is used to provide VPN services on Android. Install the [WireGuard VPN Client](https://play.google.com/store/apps/details?id=com.wireguard.android). Import the corresponding `wireguard/<name>.conf` file to your device, then setup a new connection with it. See the [Android setup instructions](/docs/client-android.md) for more detailed walkthrough.
 
-### Windows 10
+### Windows
 
-Copy your PowerShell script `windows_{username}.ps1` to the Windows client and run the following command as Administrator to configure the VPN connection.
-```
-powershell -ExecutionPolicy ByPass -File windows_{username}.ps1 -Add
-```
-
-For a manual installation, see the [Windows setup instructions](/docs/client-windows.md).
+WireGuard is used to provide VPN services on Windows. Install the [WireGuard VPN Client](https://www.wireguard.com/install/#windows-7-8-81-10-2012-2016-2019). Import the corresponding `wireguard/<name>.conf` file to your device, then setup a new connection with it.
```

Comment:
If a user is not familiar with algo, and does not know that algo generates configs, the `Import the corresponding` sentence might be a little confusing, since they might not know where the wireguard `.conf` should come from. I think it would be good to copy the phrasing from the `Apple Devices` section, which is more clear.